tests/ui/test_validation_ui.py

```diff
@@ -56,3 +64,35 @@ def test_odf_storagesystems_ui(self, setup_ui_class):
         """
         validation_ui_obj = ValidationUI(setup_ui_class)
         validation_ui_obj.odf_storagesystems_ui()
+
+    @ui
+    @tier1
+    @brown_squad
```

Comment:
Should this fall under Brown squad?

Reply:
@am-agrawa any suggestion on the squad?

Reply:
> @am-agrawa any suggestion on the squad?

It's part of dashboard validation, so I think we can keep it under Black Squad.
@ebenahar Thoughts pls?

Reply:
yes, let's keep it under black squad